locales/en/app.ftl

```diff
@@ -854,14 +854,16 @@ ad-unit-6-before-you-complete = Before you complete that next signup, use an ema
 
 -brand-firefox = Firefox
 -brand-monitor = Monitor
--brand-fx-monitor = Firefox Monitor
+-brand-fx-monitor = Mozilla Monitor
```

Comment:
I think we have possibly one more rogue reference in src/app/(nextjs_migration)/(guest)/breaches/page.tsx:28 that should change to `"brand-mozilla-monitor"`:

```sh
git grep -in "\-fx-" src | cat

src/app/(nextjs_migration)/(guest)/breaches/page.tsx:28:      siteName: l10n.getString("brand-fx-monitor"),
```

Reply:
That should be OK. We can use the new term going forward, the important thing is not to invalidate all existing translations.